calyx-py/calyx/builder_util.py

```diff
@@ -89,14 +86,13 @@ def insert_gt(comp: cb.ComponentBuilder, left, right, cellname, width):
 
     Returns handles to the cell and the combinational group.
     """
-    gt_cell = comp.gt(cellname, width)
-    return insert_comb_group(comp, left, right, gt_cell, f"{cellname}_group")
+    return insert_comb_group(comp, left, right, comp.gt(width, cellname))
 
 
-def insert_add(comp: cb.ComponentBuilder, left, right, cellname, width):
-    """Inserts wiring into component {comp} to compute {left} + {right}.
+def insert_add(comp: cb.ComponentBuilder, left, right, width, cellname=None):
+    """Inserts wiring into component {comp} to check if {left} > {right}.
 
-    <cellname> = std_add(<width>);
+    <cellname> = std_gt(<width>);
```

Comment:
This doesn't seem quite right—I'm pretty sure this function is building an add, not a comparison.